### PR TITLE
Fixed #716 - The quick create labels at top are not responsive to change

### LINKED ISSUE #716: The quick create labels at top are not responsive to change
The quick create labels at top are not responsive to change when i change say the module contact to be persons module. quick create will still say contact

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

The quick create labels at top are not responsive to change when i change say the module contact to be persons module. quick create will still say contact

## Motivation and Context

Issue reference: #716 

## How To Test This

1. Check that the quick create drop-down shows the correct values when changing a module name.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: themes/Suite7/tpls/_globalLinks.tpl [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+/**
+ *
  * SugarCRM Community Edition is a customer relationship management program developed by
  * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
-
- * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
- * Copyright (C) 2011 - 2014 Salesagility Ltd.
+ *
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by SalesAgility Ltd.
+ * Copyright (C) 2011 - 2017 SalesAgility Ltd.
```
Comment (daniel-samson) [inline themes/Suite7/tpls/_globalLinks.tpl]: license header needs to be be updated

File: themes/SuiteP/tpls/_headerModuleList.tpl [hunk 2]
```diff
@@ -5,7 +5,7 @@
  * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
  *
  * SuiteCRM is an extension to SugarCRM Community Edition developed by SalesAgility Ltd.
- * Copyright (C) 2011 - 2016 SalesAgility Ltd.
+ * Copyright (C) 2011 - 2017 SalesAgility Ltd.
```
Comment (daniel-samson) [inline themes/SuiteP/tpls/_headerModuleList.tpl]: License header needs updated

File: themes/SuiteR/tpls/_headerModuleList.tpl [hunk 3]
```diff
@@ -1,8 +1,12 @@
 {*
-/*********************************************************************************
+/**
+ *
  * SugarCRM Community Edition is a customer relationship management program developed by
  * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
  *
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by SalesAgility Ltd.
+ * Copyright (C) 2011 - 2017 SalesAgility Ltd.
```
Comment (daniel-samson) [inline themes/SuiteR/tpls/_headerModuleList.tpl]: license header needs updated

**Comment by daniel-samson** [comment]
Since we are now supporting just SuiteP - is this pull request still valid?

**Comment by mattlorimer** [comment]
Yes 7.8 supports SuiteR and Suite7

**Comment by e-reeley** [comment]
Assessed

**Pull request merged**